graylog2-server/src/main/java/org/graylog2/plugin/streams/Stream.java

```diff
@@ -50,8 +50,6 @@ public static MatchingType valueOfOrDefault(String name) {
 
     String getContentPack();
 
-    Collection<AlertCondition> getAlertConditions();
```

Comment:
✅ Works for me. At least it'll be obvious for plugin authors and users when stuff breaks.

But please add this to the `UPGRADING.rst` file.